comp/core/workloadmeta/impl/cached_entity.go

```diff
@@ -9,6 +9,7 @@ import (
 	"reflect"
 	"sort"
 
+	wmdef "github.com/DataDog/datadog-agent/comp/core/workloadmeta/def"
```

Comment:
For consistency I would alias the import to `workloadmeta`

Reply:
There is already a type `workloadmeta` defined in the package, the import would conflict with it (and I didn't want to rename it everywhere)